packages/babel-plugin-transform-class-properties/src/index.js

```diff
@@ -42,6 +42,8 @@ export default function ({ types: t }) {
   return {
     inherits: syntaxClassProperties,
 
+    capabilities: ["classProperties"],
```

Comment:
Opt-in so 3rd party plugins can also provide capabilities/dependencies. Right now it's just a string but maybe it should be a versioned number?